<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeshBaseTypeBoxWidget</class>
 <widget class="QWidget" name="MeshBaseTypeBoxWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox_Geometry">
       <property name="title">
        <string>Geometry</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_BasePoint">
          <property name="text">
           <string>Base Point:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="lineEdit_BasePoint1">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_BasePoint2">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_BasePoint3">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_Dimensions">
          <property name="text">
           <string>Dimensions:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="lineEdit_Dimensions1">
            <property name="text">
             <string>10</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_Dimensions2">
            <property name="text">
             <string>10</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_Dimensions3">
            <property name="text">
             <string>10</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0" colspan="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>254</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="pushButton_AutoSize">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>AutoSize</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox_Mesh">
       <property name="title">
        <string>Mesh</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_Division">
          <property name="text">
           <string>Division:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="lineEdit_Division1">
            <property name="text">
             <string>10</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_Division2">
            <property name="text">
             <string>10</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_Division3">
            <property name="text">
             <string>10</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_Dimensions_2">
          <property name="text">
           <string>Grading:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLineEdit" name="lineEdit_Grading1">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_Grading2">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_Grading3">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0" colspan="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>246</width>
            <height>16</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="pushButton_2">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="groupBox_Boundaries">
       <property name="title">
        <string>Boundaries</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item row="0" column="0">
         <widget class="QWidget" name="widget_X0" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="GUI::CompBaseBoundaryLabel" name="label_X0">
             <property name="text">
              <string>X-</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="GUI::CompBaseBoundaryLineEdit" name="lineEdit_X0">
             <property name="minimumSize">
              <size>
               <width>210</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>210</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QWidget" name="widget_X1" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="GUI::CompBaseBoundaryLabel" name="label_X1">
             <property name="text">
              <string>X+</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="GUI::CompBaseBoundaryLineEdit" name="lineEdit_X1">
             <property name="minimumSize">
              <size>
               <width>210</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>210</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QWidget" name="widget_Y0" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="GUI::CompBaseBoundaryLabel" name="label_Y0">
             <property name="text">
              <string>Y-</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="GUI::CompBaseBoundaryLineEdit" name="lineEdit_Y0">
             <property name="minimumSize">
              <size>
               <width>210</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>210</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QWidget" name="widget_Y1" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="GUI::CompBaseBoundaryLabel" name="label_Y1">
             <property name="text">
              <string>Y+</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="GUI::CompBaseBoundaryLineEdit" name="lineEdit_Y1">
             <property name="minimumSize">
              <size>
               <width>210</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>210</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QWidget" name="widget_Z0" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="GUI::CompBaseBoundaryLabel" name="label_Z0">
             <property name="text">
              <string>Z-</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="GUI::CompBaseBoundaryLineEdit" name="lineEdit_Z0">
             <property name="minimumSize">
              <size>
               <width>210</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>210</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QWidget" name="widget_Z1" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="GUI::CompBaseBoundaryLabel" name="label_Z1">
             <property name="text">
              <string>Z+</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="GUI::CompBaseBoundaryLineEdit" name="lineEdit_Z1">
             <property name="minimumSize">
              <size>
               <width>210</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>210</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GUI::CompBaseBoundaryLabel</class>
   <extends>QLabel</extends>
   <header>CompBaseBoundary.h</header>
  </customwidget>
  <customwidget>
   <class>GUI::CompBaseBoundaryLineEdit</class>
   <extends>QLineEdit</extends>
   <header>CompBaseBoundaryLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
